pkg/app/mssql.go

```diff
@@ -0,0 +1,351 @@
+	if err != nil {
+		return err
+	}
+	log.Print("Secret created successfully", field.M{"app": m.name, "secret": secret.Name})
+	m.secret = secret
+
+	pvcObj, err := m.getPVCObj()
+	if err != nil {
+		return err
+	}
+	pvc, err := m.cli.CoreV1().PersistentVolumeClaims(namespace).Create(ctx, pvcObj, metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+	log.Print("PVC created successfully", field.M{"app": m.name, "pvc": pvc.Name})
+	m.pvc = pvc
+
+	deploymentObj, err := m.getDeploymentObj()
+	if err != nil {
+		return err
+	}
+	deployment, err := m.cli.AppsV1().Deployments(namespace).Create(ctx, deploymentObj, metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+	log.Print("Deployment created successfully", field.M{"app": m.name, "deployment": deployment.Name})
+	m.deployment = deployment
+
+	serviceObj, err := m.getServiceObj()
+	if err != nil {
+		return err
+	}
+	service, err := m.cli.CoreV1().Services(namespace).Create(ctx, serviceObj, metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+	log.Print("Service created successfully", field.M{"app": m.name, "service": service.Name})
+	m.service = service
+
+	return nil
+}
+
+func (m *MssqlDB) IsReady(ctx context.Context) (bool, error) {
+	log.Print("Waiting for the mssql application to be ready.", field.M{"app": m.name})
+	ctx, cancel := context.WithTimeout(ctx, mssqlWaitTimeout)
+	defer cancel()
+
+	err := kube.WaitOnDeploymentReady(ctx, m.cli, m.namespace, m.deployment.Name)
+	if err != nil {
+		return false, err
+	}
+	log.Print("Application instance is ready.", field.M{"app": m.name})
+	return true, nil
+}
+
+func (m *MssqlDB) Object() crv1alpha1.ObjectReference {
+	return crv1alpha1.ObjectReference{
+		Kind:      "deployment",
+		Name:      "mssql-deployment",
+		Namespace: m.namespace,
+	}
+}
+
+func (m *MssqlDB) Uninstall(ctx context.Context) error {
+	err := m.cli.AppsV1().Deployments(m.namespace).Delete(ctx, m.deployment.Name, metav1.DeleteOptions{})
+	if err != nil {
+		return err
+	}
+	log.Print("Deployment deleted successfully", field.M{"app": m.name})
+
+	err = m.cli.CoreV1().PersistentVolumeClaims(m.namespace).Delete(ctx, m.pvc.Name, metav1.DeleteOptions{})
+	if err != nil {
+		return err
+	}
+	log.Print("PVC deleted successfully", field.M{"app": m.name})
+
+	err = m.cli.CoreV1().Services(m.namespace).Delete(ctx, m.service.Name, metav1.DeleteOptions{})
+	if err != nil {
+		return err
+	}
+	log.Print("Service deleted successfully", field.M{"app": m.name})
+
+	err = m.cli.CoreV1().Secrets(m.namespace).Delete(ctx, m.secret.Name, metav1.DeleteOptions{})
+	if err != nil {
+		return err
+	}
+	log.Print("Secret deleted successfully", field.M{"app": m.name})
+	return nil
+}
+
+func (m *MssqlDB) Ping(ctx context.Context) error {
+	log.Print("Pinging mssql database", field.M{"app": m.name})
+	count := "/opt/mssql-tools/bin/sqlcmd -S localhost -U " + dbUserName + " -P \"" + dbPass + "\" -Q " +
```

Comment:
wouldn't just `sqlcmd` work, instead of specifying absolute path.

Reply:
I tried with just `sqlcmd` command but it didn't work that's why I had to use absolute path.